### DISCUSSION TITLE
Why my logs werent sent to otel-collector but trace were?

### DISCUSSION THREAD

**Discussion by Wudadada** [open-telemetry/opentelemetry-go#5895] [Q&A] [answered]
I can saw the trace data but log data, is there something wrong with my code | config or it is a bug?
my init code
<img width="587" alt="image" src="https://github.com/user-attachments/assets/742ea0ef-52e5-4dd0-b06d-68dbfd55362f">
<img width="808" alt="image" src="https://github.com/user-attachments/assets/e5a3bc2f-1566-47ee-8e5a-760fcf99b639">
<img width="575" alt="image" src="https://github.com/user-attachments/assets/56b292b7-1934-4d94-9d3e-5f28c09c9a5e">

my business code
<img width="441" alt="image" src="https://github.com/user-attachments/assets/ffd5ed55-5882-4d7f-b207-6ffd934dfc65">
<img width="843" alt="image" src="https://github.com/user-attachments/assets/13fd54bf-de43-42f8-b3ee-d3f1cd0e5faf">

my log correctly ouput
<img width="906" alt="image" src="https://github.com/user-attachments/assets/91945e2b-b379-4330-8357-1085242e6cd1">

my opentelemetry-collector configration
<img width="482" alt="image" src="https://github.com/user-attachments/assets/b35b68e9-e87c-4242-a266-18a816ec6ec9">

**Comment by pellared** [comment, marked as answer]
You need to use https://pkg.go.dev/go.opentelemetry.io/contrib/bridges/otelslog if you want `slog.Logger` to send logs to OpenTelemetry Logger. Currently you are using `slog.NewTextHandler` to send them to standard output.

You can also look at https://opentelemetry.io/docs/languages/go/getting-started/#add-custom-instrumentation

PS. Share text instead of printscreens.

**Reply by Wudadada** [reply]
Thanks a lot
